python/api/src/araali/quickstart_bend.py

```diff
@@ -0,0 +1,47 @@
+from . import utils
```

Comment:
why is the file quickstart_bend.py? It is not implementing quickstart_bend. Maybe quickstart_api.py, or quickstart.py (since the whole thing is api anyways)